rasa/core/channels/botframework.py

```diff
@@ -188,6 +200,37 @@ def __init__(self, app_id: Text, app_password: Text) -> None:
         self.app_id = app_id
         self.app_password = app_password
 
+        self.jwt_keys: Dict[Text, Any] = {}
+        self.jwt_update_time = datetime.datetime.fromtimestamp(0)
+
+        self._update_cached_jwk_keys()
+
+    def _update_cached_jwk_keys(self) -> None:
+        if datetime.datetime.now() - self.jwt_update_time < datetime.timedelta(days=1):
```

Comment:
For now, I'd say a daily update of the JWT keys is plenty, and also currently no need to make this configurable.